dm/dm/config/task_converters.go

```diff
@@ -53,6 +53,7 @@ func TaskConfigToSubTaskConfigs(c *TaskConfig, sources map[string]DBConfig) ([]*
 		cfg.Timezone = c.Timezone
 		cfg.Meta = inst.Meta
 		cfg.CollationCompatible = c.CollationCompatible
+		cfg.Experimental.AsyncCheckpointFlush = true
```

Comment:
Why the right side of equality operator is not `c.Experimental.AsyncCheckpointFlush`. Checkpoint flush will be in async mode in this situation.